lightbluetent/templates/society/welcome.html

```diff
@@ -96,12 +96,21 @@ <h6 class="card-title">Day {{n}}</h6>
                 {% endfor %}
 
                 <h6 class="card-title text-muted">Join the live session</h6>
-                <form>
+
+                <form method="post">
+                    <input type="hidden" name="_csrf_token" value="{{ csrf_token() }}">
+
                     <div class="form-group">
-                        <input type="name" class="form-control" id="name" placeholder="Enter your name..." disabled>
+                        <input type="name" class="form-control {% if errors.full_name is defined %} is-invalid{% endif %}"
```

Comment:
Does the name persist across different stalls? If not, what do you think about this?

Reply:
There is no persistence. I would be inclined towards allowing users to choose their names per-stall, but open to other thoughts on this.

Reply:
Agree